### PR TITLE
Add monaco as cell input

### PR DESCRIPTION
**Pull request by tengpeng**

### CODE REVIEW

**Comment by tengpeng** [comment]
@oliyg

File: packages/browser/src/components/input.tsx [hunk 1]
```diff
@@ -19,12 +20,13 @@ export const Input: React.FC<Props> = ({ cellVM, onInputChange, onKeyDown }) =>
     }
 
     const renderCodeInput = () => {
-        return <textarea
-            style={{ width: '100%', minHeight: "90px", backgroundColor: "#404040" }}
-            value={cellVM.cell.source}
-            // placeholder="code"
-            onKeyDown={onKeyDown}
-            onChange={(event) => onInputChange(event, cellVM)} ></textarea>
+        return  <Editor />
```
Comment (tengpeng) [inline packages/browser/src/components/input.tsx:23]: I will remove this.

File: packages/browser/yarn.lock [hunk 2]
```diff
@@ -1856,6 +1856,14 @@
     "@types/prop-types" "*"
     csstype "^2.2.0"
 
+"@types/react@^16.x":
```
Comment (tengpeng) [inline packages/browser/yarn.lock:1859]: I am not sure if we should keep yarn.lock file in git repo.

File: packages/browser/package.json [hunk 3]
```diff
@@ -23,6 +23,7 @@
     "highlight.js": "^9.18.1",
```
Comment (tengpeng) [inline packages/browser/package.json:23]: TODO: clean up package.json

File: packages/browser/yarn.lock (hunk 2 above)
Comment (oliyg) [reply packages/browser/yarn.lock:1859]: > the yarn.lock file will ensure that they get precisely the same dependencies as you have.

We should keep yarn.lock file in git repo.

**Pull request merged**